pkg/descriptions.json

```diff
@@ -1204,6 +1204,14 @@
         ]
     },
     {
+        "Name": "System.Runtime.Serialization.Formatters",
+        "Description": "Provides common types for libraries that support binary serialization.",
```

Comment:
Correct. I meant to use the name of this technology. I thought we call it binary serialization. Should I call it "runtime serialization"?